tracks/create-actions.es6

```diff
@@ -122,20 +140,42 @@ export default function createActions(nodes, mountPoint='tracks') {
     }
   }
 
+  const UNMUTE = `${mountPoint}/UNMUTE`;
+  function unMute(id) {
+    return function unMuteThunk(dispatch, getState) {
+      const track = dotKey(mountPoint, getState())[id];
+
+      if (track && track.isReady && track.isPlaying) {
+        nodes.unmute(id);
```

Comment:
little thing but if this one is `unmute` shouldn't the action create be `unmute` too instead of `unMute`?